test/docker/tutorial/instructions.md

```diff
@@ -1,22 +1,54 @@
 # Efficient In-Memory Inverted Indexes
 
-## Running PISA in container
+## Prerequisites
+
+### Download input data
+
+TODO
+
+### Download container image
+
+First,
+[download the image](https://sigir2025.it-mil-1.linodeobjects.com/pisa-tutorial.tar.gz).
```

Comment:
Where is this hosted currently? I need to update this once the image is finalized, correct?